lib/livebook/teams.ex

```diff
@@ -223,6 +281,19 @@ defmodule Livebook.Teams do
     add_errors(change(secret), Secret.__schema__(:fields), errors_map)
   end
 
+  defp add_file_system_errors(%FileSystem.S3{} = file_system, errors_map) do
+    errors_map =
+      for {key, values} <- errors_map, into: %{} do
+        case key do
+          "name" -> {"bucket_url", values}
+          "type" -> {"bucket_url", values}
+          "value" -> {"secret_access_key", values}
+        end
```

Comment:
What about the unique constraint?

Reply:
We already have it on Teams side

Reply:
Right, and the mapping here is to translate the teams api error, what am I missing? :D

Reply:
I would remove the unique constant in `name` from Teams, unless Livebook itself does not like duplicate bucket_urls? But I think it doesn't care?

Reply:
`id` is based on bucket url, so having many within the same team would be an issue yeah. And having both in the UI is also weird, which is which (well both are the same)?

Reply:
Could be both the same, but with different AWS credentials, which could have different control access

Reply:
> Right, and the mapping here is to translate the teams api error, what am I missing? :D

Yes, I'm mapping the error keys to struct fields, just to make sure we're showing the errors in the right field, but some of them we don't have on the livebook side

Reply:
Right, but I don't think having many within the same team since the point is to share.

If we distinguished by credentials, then imagine I have a notebook where I add file links to and S3 bucket (that I added to my personal hub). Now if someone else opens that notebook and they have the same S3 bucket, but with different credentials, it wouldn't match.

Reply:
Makes sense